plugins/woocommerce/tests/e2e-pw/fixtures/wc-install.setup.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Internal dependencies
+ */
+import { test as setup } from './fixtures';
+import { ADMIN_STATE_PATH } from '../playwright.config';
+
+setup.use( { storageState: ADMIN_STATE_PATH } );
+
+setup( 'install WC using WC Beta Tester', async ( { page } ) => {
+	if ( process.env.INSTALL_WC === 'true' ) {
```

Comment:
May I suggest to use `setup.skip()` instead of this conditional? With the skip option the setup script will be reported as such and it will be more clear that the plugins was not installed.

Something in the lines of:
```
setup.skip( !process.env.INSTALL_WC, 'Skipping installing the plugin, INSTALL_WC not found');
```

Reply:
Thanks Adi! This is nice and clean 👍 Implemented it here [b720af5](https://github.com/woocommerce/woocommerce/pull/55063/commits/b720af509827e87a5430e0ae656112afbc923c49)

Reply:
This is now part of [`site.setup.js` fixture](https://github.com/woocommerce/woocommerce/pull/55063/files#diff-f6a4141a21bfc84664a41827dcf301bc2facc6822d2d1d56ec5eaf6ce6bb5188R109-R112).